packages/react-components/react-drawer/src/components/DrawerBody/DrawerBody.cy.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+import type {} from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { webLightTheme } from '@fluentui/react-theme';
+import { Drawer, DrawerBody } from '@fluentui/react-drawer';
+
+const mountFluent = (element: JSX.Element) => {
+  mount(<FluentProvider theme={webLightTheme}>{element}</FluentProvider>);
+};
+
+function assertScrollPosition(element: HTMLElement, position: number) {
+  expect(element.scrollTop).to.equal(position);
+}
+
+describe('DrawerBody', () => {
+  it('render drawer body component', () => {
```

Comment:
can we follow human readable spec descriptions ?

```suggestion
  it('should render drawer body component', () => {
```